mobius-android/src/main/java/com/spotify/mobius/android/MobiusLoopViewModel.java

```diff
@@ -129,6 +147,49 @@ protected MobiusLoopViewModel(
         maxEffectsToQueue);
   }
 
+  /**
+   * Creates a new MobiusLoopViewModel instance with a default maximum effect queue size.
+   *
+   * @param loopFactoryProvider The provider for the factory, that gets passed all dependencies
+   * @param modelToStartFrom The initial model for the loop
+   * @param init the {@link Init} function of the loop
+   * @param <M> the model type
+   * @param <E> the event type
+   * @param <F> the effect type
+   * @param <V> the view effect type
+   */
+  public static <M, E, F, V> MobiusLoopViewModel<M, E, F, V> create(
+      @Nonnull MobiusLoopFactoryProvider<M, E, F, V> loopFactoryProvider,
+      @Nonnull M modelToStartFrom,
+      @Nonnull Init<M, F> init) {
+    return create(loopFactoryProvider, modelToStartFrom, init, 100);
+  }
+
+  /**
+   * Creates a new MobiusLoopViewModel instance.
+   *
+   * @param loopFactoryProvider The provider for the factory, that gets passed all dependencies
+   * @param modelToStartFrom the initial model for the loop
+   * @param init the {@link Init} function of the loop
+   * @param maxEffectsToQueue the maximum number of effects to queue while paused
+   * @param <M> the model type
+   * @param <E> the event type
+   * @param <F> the effect type
+   * @param <V> the view effect type
+   */
+  public static <M, E, F, V> MobiusLoopViewModel<M, E, F, V> create(
```

Comment:
what's the purpose of these two functions? you won't be able to use them when subclassing, and subclassing is required since the class is used as a key in ViewModelStore

Reply:
You can use these methods and the view model without subclassing it if you write a View Model Factory for each fragment you need it for. That's been one approach I've been using, and it's pretty easy and straight forward to write with Kotlin. (I'm working on a wiki update and sample project which will show this too)